src/commands/prs.js

```diff
@@ -16,6 +16,10 @@ exports.args = [
 			if (!templates.title) {
 				return 'Please provide a Pull Request title'
 			}
+			if (!templates.body) {
+				return 'Please provide a Pull Request body'
```

Comment:
If the user provides neither a title nor a body, the message they'll get is `Please provide a Pull Request title`. They'll do that, then be hit with `Please provide a Pull Request body`. That's not a good UX. The message should take both into account if both are missing, following the [pattern from the `project` operation](https://github.com/Financial-Times/nori/blob/master/src/commands/project.js#L17-L26)

Reply:
Fixed